### PR TITLE
Correctly configure server certificate authority.

### PR DESCRIPTION
**Pull request by awong-dev**
With this, NODE_TLS_REJECT_UNAUTHORIZED=0 should no longer be required
to establish an encrypted connection with SOAP endpoints using CAs that
are "not generally trusted" (which includes all Federal CA and agency
internal CAs).

This is important for mitigating MiTM attacks on establishing the TLS
connection.

### CODE REVIEW

**Comment by joesoftware** [comment]
LGTM. (Inspected and tested locally with my copy of the private key.)

**Comment by robbiethegeek** [comment]
@awong-dev can you update the README.md to describe the requirements for local key and crt files.  If we need to manually create can you describe where they come from.

You are close to it and I feel like I can't wrap my head around what is still required or not.

**Comment by awong-dev** [comment]
Back at ya Robbie

File: README.md [hunk 1]
```diff
@@ -58,227 +58,135 @@ npm install
...
+staging and production servers. During normal development, test against `dev`
+or `sqa`.
+
+##### Selecting an ES endpoint
+
+The server endpoints are configured in `config.json` via the
+`config.soap.endpoint` and `config.soap.wsdl` keys.  See the `endpoint` object
+for a list of the common server endpoints to use.
+
+The configuration default s https SOAP request sent to `esDev`. By default,
```
Comment (markolson) [inline README.md]: "default is" ?

Comment (awong-dev) [inline README.md]: fixed.

**Comment by robbiethegeek** [comment]
Thanks @awong-dev LGTM will merge when the tests pass.

**Pull request merged**